Abraham development (#1)
abrahamharos/abrahamharos-step-2020: README.md

# Google Student Training in Engineering Program

> This repo contains abraham's portfolio and projects of the first half of STEP.

Reviewer: ```suggestion
This repo contains Abraham's portfolio and projects for ...
```
(Capitalize your first name and change "of" to "for")
